joplin/templates/wagtailadmin/shared/header.html

```diff
@@ -0,0 +1,51 @@
+{% load i18n %}
+{% comment %}
+
+    Variables accepted by this template:
+
+    title
+    subtitle
+
+    search_url - if present, display a search box. This is a URL route name (taking no parameters) to be used as the action for that search box
+    query_parameters - a query string (without the '?') to be placed after the search URL
+
+    icon - name of an icon to place against the title
+
+    tabbed - if true, add the classname 'tab-merged'
+    merged - if true, add the classname 'merged'
+
+    add_link - if present, display an 'add' button. This is a URL route name (taking no parameters) to be used as the link URL for the button
+    add_text - text for the 'add' button
+{% endcomment %}
+<header class="{% if merged %}merged{% endif %} {% if tabbed %}tab-merged{% endif %} {% if search_form %}hasform{% endif %}" role="banner">
+    {% block breadcrumb %}{% endblock %}
+    <div class="row nice-padding">
+        <div class="left">
+            <div class="col header-title">
+                <h1 class="icon icon-{{ icon }}">
+                {{ title }}{% if subtitle %} <span>{{ subtitle }}</span>{% endif %}</h1>
+            </div>
+            {% comment %}
```

Comment:
Any reason we have this commented out instead of just removing it completely?

Reply:
I'm not 100% opposed to ditching it, I think its helpful to keep it as a reference for maintainability. Going back and forth between repos is kind of a pain, and its hard to see the diffs between vanilla wagtail and the changes we make and why.